src/screens/setting/components/index.js

```diff
@@ -657,10 +657,10 @@ const Setting = props => {
                           right={props => (
                             <Switch
                               trackColor={{false: '#767577', true: '#32a32e'}}
-                              thumbColor={memberEnabled ? 'white' : 'white'}
+                              thumbColor={regEvents ? 'white' : 'white'}
```

Comment:
what's the use if we ar eusing same color in all condition? is the ternary operator necessary